### DISCUSSION TITLE
Sass quiet options?

### DISCUSSION THREAD

**Discussion by gnimmelf** [glromeo/esbuild-sass-plugin#50] [Ideas]
Would it be an idea to implement passing on the `quiet` option(s) to suppress warnings?

https://sass-lang.com/documentation/cli/dart-sass#quiet

Usecase is less output in pipeline.

Cheers!

**Comment by glromeo** [comment]
Hi @gnimmelf all [the options that the Javascript SASS API supports](https://sass-lang.com/documentation/js-api/interfaces/LegacyFileOptions) are accepted and passed over by the plugin.
e.g.
```
sassPlugin({
  quietDeps: true
})
```
If you still see warnings bear in mind that there are some caveats... you can read more about them here: #17 
Maybe you can pass a [logger](https://sass-lang.com/documentation/js-api/interfaces/LegacySharedOptions#logger) if you want more control over the output.
```
sassPlugin({
  logger: {
    warn(message, opts) {
      // totally silent!
    },
    debug(message, opts) {
      // totally silent!
    },
  }
})
```

**Reply by glromeo** [reply]
by the way...I am working on a new version since SASS released [new APIs](https://sass-lang.com/documentation/js-api)

**Reply by gnimmelf** [reply]
Hi again, 

thanks for the feedback. 

Passing a logger perfectly solved the problem at hand, as I have no control over the sass-internal imports. 

It is really a good time of api-consolidations these days! Keep it up, really appreciated!

Cheers,
Flemming
